api/views.py

```diff
@@ -71,6 +71,12 @@ def create(self, request, **kwargs):
                 transaction.on_commit(
                     lambda: AnswerViewSet.__update_meta_sheets_for_feeder(puzzle)
```

Comment:
Come to think of it -- this line should also be wrapped by `if google_api_lib.enabled()`.

Reply:
Aha hmm-- `update_meta_sheets_for_feeder` used to have a check for this inside and it looks like as part of #295 @rawxfish moved the check inside the GoogleApiClientTask __init__ in google_api_lib/utils.py.

Reply:
Testing locally I can confirm that the celery worker complains so I'll just fix on this branch.

Reply:
Split it off for clarity: #338

Reply:
Whoops sorry about that